addon/-private/system/store.js

```diff
@@ -342,7 +342,7 @@ Store = Service.extend({
   createRecord(modelName, inputProperties) {
     assert(`You need to pass a model name to the store's createRecord method`, isPresent(modelName));
     assert(`Passing classes to store methods has been removed. Please pass a dasherized string instead of ${modelName}`, typeof modelName === 'string');
-    let normalizedModelName = this._classKeyFor(modelName);
+    let normalizedModelName = normalizeModelName(modelName);
```

Comment:
@runspired I believe you recently introduced `_classKeyFor` as a wrapper for `normalizeModelName`. Are you ok with changing this back to just `normalizeModelName`?